web/src/app/util/CopyText.tsx

```diff
@@ -22,19 +22,37 @@ interface CopyTextProps {
   placement?: TooltipProps['placement']
   title?: string
   value: string
+  textOnly?: boolean
 }
 
 export default function CopyText(props: CopyTextProps): JSX.Element {
   const classes = useStyles()
-  const [showTooltip, setShowTooltip] = useState(false)
+  const [copied, setCopied] = useState(false)
 
-  return (
-    <Tooltip
-      onClose={() => setShowTooltip(false)}
-      open={showTooltip}
-      title='Copied!'
-      placement={props.placement || 'right'}
-    >
+  let content
+  if (props.textOnly) {
+    content = (
+      <span
```

Comment:
`Typography`? It should be able to handle root level html tags for `span`

Reply:
It already is rendered in/wrapped in Typography through the `ListItemText` component.

As far as here in `CopyText` -- `<span>` was used so it just inherits the styles and can have event handlers attached -- trying to follow the same behavior as `<AppLink>`, which uses a base `<a>` tag when `textOnly` is unset. That way it relies on it's container for any styling and with or without a link can be put into whatever container is needed.